<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera</class>
 <widget class="QWidget" name="camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>camera</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 91);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="flip_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 255, 34);
color: rgb(30, 11, 135);
font: 24pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Mirror</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="horizontal_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 255, 34);
color: rgb(30, 11, 135);
font: 24pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Flip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="el_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 255, 34);
color: rgb(30, 11, 135);
font: 20pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>eyes - lip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="emoji_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 255, 34);
color: rgb(30, 11, 135);
font: 24pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Emoji</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="checkered_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 255, 34);
color: rgb(30, 11, 135);
font: 24pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Checkered </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="video_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>641</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 255, 16);
font: 125 60pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Select a mode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
